### ISSUE TITLE
release the .apk on github so obtainium can access it 

### ISSUE THREAD

**Issue by RustoMCSpit** [pixelfed/mobile-app#148]

**Comment by amiuhle** [comment]
You can add the RSS feed: https://pixelfed.org/mobile-apps/pixelfed-for-android/builds.rss

**Comment by Danie10** [comment]
I was looking for exactly this too. I already tried the RSS feed (and versions of it) in Obtanium, but it cannot parse the RSS feed. A request was put in for Obtanium to try parse the Pixelfed webpage, but the reasons given are Obtanium needs Javascript to do that. See explanation at [Obtanium #454](https://github.com/ImranR98/Obtainium/issues/454).

The simple and easiest for now would be just to put the beta APK releases on this Github project. It is just a simple upload under releases, and that would really help many out. If issues can be logged here, I hope the releases could also be here?

**Comment by amiuhle** [comment]
@Danie10 Yes, it works for me by just adding the RSS URL as an App Source URL in Obtainium. It gives a warning about a mismatch of the domain and the app package, but when I continue I'm able to install the apk.

I agree that adding the APK on GitHub releases would be the best solution though, my comment was meant as a workaround.

**Comment by Danie10** [comment]
@amiuhle thanks for confirming. Seems the beta has finally hit the Play Store, so maybe that is an easier way for most. It's been a very long wait ;-)

**Comment by EdoSecco** [comment]
The RSS hasn't been working for some days now, is there any problem?
